master/Makefile

```diff
@@ -88,25 +88,7 @@ stream-gen: $(STREAM_PYTHON_CLIENT) $(STREAM_TS_CLIENT)
 mocks: build/mock_gen.stamp
 
 build/mock_gen.stamp: $(MOCK_INPUTS)
-	mockery --quiet --name=DB --dir=internal/db --output internal/mocks --filename db.go
-	mockery --quiet --name=Resources --dir=internal/sproto --output internal/mocks --filename resources.go
-	mockery --quiet --name=UserAuthZ --dir=internal/user --output internal/mocks --filename user_authz_iface.go
-	mockery --quiet --name=WorkspaceAuthZ --dir=internal/workspace --output internal/mocks --filename workspace_authz_iface.go
-	mockery --quiet --name=ProjectAuthZ --dir=internal/project --output internal/mocks --filename project_authz_iface.go
-	mockery --quiet --name=ExperimentAuthZ --dir=internal/experiment --output internal/mocks --filename authz_experiment_iface.go
-	mockery --quiet --name=NSCAuthZ --dir=internal/command --output internal/mocks --filename nsc_authz_iface.go
-	mockery --quiet --name=ModelAuthZ --dir=internal/model --output internal/mocks --filename authz_model_iface.go
-	mockery --quiet --name=PodInterface --srcpkg=k8s.io/client-go/kubernetes/typed/core/v1 --output internal/mocks --filename pod_iface.go
-	mockery --quiet --name=EventInterface --srcpkg=k8s.io/client-go/kubernetes/typed/core/v1 --output internal/mocks --filename event_iface.go
-	mockery --quiet --name=NodeInterface --srcpkg=k8s.io/client-go/kubernetes/typed/core/v1 --output internal/mocks --filename node_iface.go
-	mockery --quiet --name=JobInterface --srcpkg=k8s.io/client-go/kubernetes/typed/batch/v1 --output internal/mocks --filename job_iface.go
-	mockery --quiet --name=ResourceManager --dir=internal/rm --output internal/mocks --filename rm.go
-	mockery --quiet --name=AllocationService --dir=internal/task --output internal/mocks/allocationmocks --filename allocation_service.go --outpkg allocationmocks
-	mockery --quiet --name=ResourceManagerAuthZ --dir=internal/rm --output internal/mocks --filename rm_authz_iface.go
-	mockery --quiet --name=Interface --output internal/mocks --srcpkg "k8s.io/client-go/kubernetes" --filename k8s_clientset.go --structname K8sClientsetInterface
-	mockery --quiet --name=CoreV1Interface --output internal/mocks --srcpkg "k8s.io/client-go/kubernetes/typed/core/v1" --filename k8s_corev1_iface.go --structname K8sCoreV1Interface
-	mockery --quiet --name=BatchV1Interface --output internal/mocks --srcpkg "k8s.io/client-go/kubernetes/typed/batch/v1" --filename k8s_batchv1_iface.go --structname K8sBatchV1Interface
-
+	mockery --config "./.mockery.yaml"
```

Comment:
is the output a lot? 

Should we add ```--quite``` to this? I am fine either way

Reply:
We do not need it as it does not add any noise:

```
% make -C master check                        
rm -f build/schema_gen.stamp
rm -f ../harness/determined/common/streams/wire.py
rm -f ../webui/react/src/services/stream/wire.ts
../tools/scripts/gen-attributions.py master packaging/LICENSE
go generate ./pkg/schemas/...
mkdir -p build
touch build/schema_gen.stamp
go build -o build/stream-gen ./cmd/stream-gen
build/stream-gen ./internal/stream//authz_basic_impl.go ./internal/stream//subscription.go ./internal/stream//models.go ./internal/stream//test_util.go ./internal/stream//model_versions.go ./internal/stream//util.go ./internal/stream//supervisor.go ./internal/stream//socketlike.go ./internal/stream//messages.go ./internal/stream//projects.go ./internal/stream//authz_iface.go ./internal/stream//publisher.go --python --output ../harness/determined/common/streams/wire.py
build/stream-gen ./internal/stream//authz_basic_impl.go ./internal/stream//subscription.go ./internal/stream//models.go ./internal/stream//test_util.go ./internal/stream//model_versions.go ./internal/stream//util.go ./internal/stream//supervisor.go ./internal/stream//socketlike.go ./internal/stream//messages.go ./internal/stream//projects.go ./internal/stream//authz_iface.go ./internal/stream//publisher.go --ts --output ../webui/react/src/services/stream/wire.ts
mockery --config "./.mockery.yaml"
mkdir -p build
touch build/mock_gen.stamp
# Checking that committed, generated code is up-to-date by ensuring that
# git reports the files as unchanged after forcibly regenerating the files:
test -z "" || (git diff; false)
go mod tidy
git diff --quiet ../go.mod ../go.sum
golangci-lint --build-tags integration run --timeout 10m
WARN [linters_context] running gomodguard failed: unable to parse module file go.mod: go.mod:3: invalid go version '1.22.0': must match format 1.23: if you are not using go modules it is suggested to disable this linter 
```